catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -506,14 +506,10 @@ protected MetacardType mergeAttributes(MetacardType metacardType) {
             .flatMap(Collection::stream)
             .collect(Collectors.toSet());
 
-    // Guard against empty collection. If the collection is empty,
-    // the MetacardTypeImpl constructor throws an exception.
-    if (!additionalAttributes.isEmpty()) {
-      returnObject =
-          new MetacardTypeImpl(metacardType.getName(), metacardType, additionalAttributes);
-    }
+    // always add extracted attributes
+    additionalAttributes.addAll(new ExtractedAttributes().getAttributeDescriptors());
```

Comment:
Should a similar change be made to the PdfInputTransformer?

Reply:
the pdf transformer does things a little differently so I don't think it needs to change.